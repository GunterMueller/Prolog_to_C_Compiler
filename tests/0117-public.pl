:- public foo, bar/1.

main :-
	recorded('$public', p(A, B, C)),
	display([A, B]), nl,
	'$call'(C, [123]),
	fail.
main.

foo(X) :- display(foo(X)), nl.
bar(X) :- display(bar(X)), nl.
