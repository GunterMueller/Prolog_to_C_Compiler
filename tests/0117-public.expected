[foo,1]
foo(123)
[bar,1]
bar(123)
